packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -3,130 +3,184 @@
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+ * @prop --calcite-panel-description-color: Specifies the color of the component's description.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
  * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
+ * @prop --calcite-panel-header-background-color: Specifies the component header's background color.
  * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+ * @prop --calcite-panel-header-color: Specifies the component header's  color.
+ * @prop --calcite-panel-header-z-index: Specifies the component header's z-index.
+ * @prop --calcite-panel-fab-z-index: Specifies the component fab's z-index.
  */
 
 :host {
   @extend %component-host;
-  @apply relative flex w-full h-full flex-auto;
-
-  --calcite-min-header-height: calc(var(--calcite-icon-size) * 3);
+  position: relative;
+  display: flex;
+  inline-size: 100%;
+  block-size: 100%;
+  flex: 1 1 auto;
 }
 
 @include disabled();
 
-@import "../../assets/styles/header";
+.header {
+  color: var(--calcite-panel-header-color, var(--calcite-color-text-2));
+  fill: var(
+    --calcite-panel-header-color,
+    var(--calcite-color-text-2)
+  ); // question: not sure if this is needed. Not sure why it was there in the first place.  https://github.com/Esri/calcite-design-system/blame/2e8abc5cea258e188445a223b15c2e469d6baef5/src/assets/styles/_header.scss
+  margin: 0;
+  display: flex;
+  align-content: space-between;
+  align-items: center;
+}
 
-.container {
-  @apply bg-background m-0 flex w-full flex-auto flex-col items-stretch p-0;
+.heading {
+  margin: 0;
+  padding: 0;
+  font-weight: var(--calcite-font-weight-medium); // question: are we doing font-weight tokens?
+}
 
-  transition:
+.header .heading {
+  flex: 1 1 auto;
+  padding: var(--calcite-spacing-xxs);
+}
+
+.container {
+  background-color: var(--calcite-panel-background-color, var(--calcite-color-background));
+  margin: 0;
+  display: flex;
+  inline-size: 100%;
+  flex: 1 1 auto;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 0;
+
+  transition: var(
+    --calcite-internal-panel-transition,
     max-block-size var(--calcite-animation-timing),
-    inline-size var(--calcite-animation-timing);
+    inline-size var(--calcite-animation-timing)
+  );
 }
 
 .container[hidden] {
-  @apply hidden;
+  display: none;
 }
 
 .header {
-  @apply bg-foreground-1 flex flex-col
-    z-header;
-  border-block-end: var(--calcite-panel-header-border-block-end, 1px solid var(--calcite-color-border-3));
+  display: flex;
+  flex-direction: column;
+  z-index: var(--calcite-panel-header-z-index, var(--calcite-z-index-header));
+  background-color: var(--calcite-panel-header-background-color, var(--calcite-color-foreground-1));
+  border-block-end: var(
```

Comment:
I don't think there are any requests from customers to have control of the border width. I'd prefer we split this rule up into `border-block-end-style`, `border-block-end-width`, and `border-block-end-color` then only add a component token for `--calcite-panel-border-color`. Then use the same pattern for other border rules within the component.

Reply:
This var was already there so cant change it.